packages/OS400/initscript.sh

```diff
@@ -278,15 +279,16 @@ get_make_vars()
 
 {
         eval "$(sed -e ': begin'                                        \
-                -e '/\\\\$/{'                                           \
+                -e '/\\$/{'                                             \
                 -e 'N'                                                  \
-                -e 's/\\\\\\n/ /'                                       \
+                -e 's/\\\n/ /'                                          \
                 -e 'b begin'                                            \
                 -e '}'                                                  \
-                -e '/^[A-Za-z_][A-Za-z0-9_]*[[:space:]]*=/!d'           \
-                -e 's/@\\([A-Za-z0-9_]*\\)@/${\\1}/g'                   \
-                -e 's/[[:space:]]*=[[:space:]]*/=/'                     \
-                -e 's/=\\(.*[^[:space:]]\\)[[:space:]]*$/=\\"\\1\\"/'   \
-                -e 's/\\\$(\\([^)]*\\))/\${\\1}/g'                      \
-                < \""${1}"\")"
+                -e 's/[[:space:]][[:space:]]*/ /g'                      \
+                -e '/^[A-Za-z_][A-Za-z0-9_]* *=/!d'                     \
+                -e 's/@\([A-Za-z0-9_]*\)@/${\1}/g'                      \
+                -e 's/ *= */=/'                                         \
+                -e 's/=\(.*[^ ]\) *$/=\"\1\"/'                          \
```

Comment:
```suggestion
                -e 's/=\(.*[^ ]\) *$/="\1"/'                            \
```
Can we save escaping the double-quotes? (it works in my limited test)

Reply:
Works on the os/400 too. Thanks for the suggestion. Force pushed.